src/constants/rules.js

```diff
@@ -0,0 +1,17 @@
+/** 로또 게임의 룰과 관련된 상수들입니다. */
+export const LOTTO = Object.freeze({
+  NUM_COUNT: 6,
```

Comment:
```suggestion
 LENGTH: 6,
```
`LOTTO_RULES.NUM_COUNT`가 로또 번호 길이를 의미하는 것 같은데 조금 더 명확하게 `LOTTO_RULES.LENGTH` 이렇게 사용해보는건 어떨까요??